app/src/main/java/org/fossasia/openevent/utils/ISO8601Date.java

```diff
@@ -31,20 +34,37 @@ public static String now() {
         return fromCalendar(GregorianCalendar.getInstance());
     }
 
-    /**
-     * Transform ISO 8601 string to Calendar.
-     */
-    public static Calendar toCalendar(final String iso8601string)
-            throws ParseException {
-        Calendar calendar = GregorianCalendar.getInstance();
-        String s = iso8601string.replace("Z", "+00:00");
+
+    public static String getTimeZoneDate(final Date date) throws ParseException {
+
+        SimpleDateFormat dateFormat = new SimpleDateFormat("EE, dd MMM yyyy, HH:mm, z");
+        dateFormat.setTimeZone(TimeZone.getDefault());
+        String DateToStr = dateFormat.format(date);
+        return DateToStr;
+    }
+
+    public static Date getDateObject(final String iso8601String) throws ParseException {
+
+        StringBuilder s = new StringBuilder();
+        s.append(iso8601String).append("Z");
+        String final1 = s.toString();
+        Log.d("time", final1);
+
+        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+        format.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
Sorry I don't understand. Alright we can have a setting for this but I don't know if we can get the "user's timestamp"